### DISCUSSION TITLE
Problem with calculating the Physiological Equivalent Temperatur (PET) in SOLWEIG

### DISCUSSION THREAD

**Discussion by danelzw** [UMEP-dev/UMEP#350] [Q&A] [answered]
I have a question about calculating the physiologically equivalent temperature (PET) in SOLWEIG. I have defined different POIs for my study area. At first I wondered why the PET in the Output folder is identical for each POI (despite different mean radiation temperature). Then i noticed that the PET is the same as the air temperature initially entered via the meteorological dataset. So if i understand correctly, no PET is recalculated. Does anyone have any idea what this could be due to? Or have I made a mistake?

Kind regards and thanks in advance!

**Comment by biglimp** [comment, marked as answer]
Hm, this might be a bug. If so, we can move this thread over to issues but first, can you please attach your input data (DSMs, SVFs, metdata, POIs and RunInfo from your previous run).

**Reply by danelzw** [reply]
Thanks for your reply! I have to ask my professor if I am allowed to share the data. But this issue is also the case for the UMEP Tutorial for Thermal comfort: https://umep-docs.readthedocs.io/projects/tutorial/en/latest/Tutorials/IntroductionToSolweig.html#introtosolweig. I did the tutorial and there the PET also equals the input air temperature data. 
I have attached the output POI file of the tutorial. When i have feedback from my professor, i can upload the other files. 
[POI_1.txt](https://github.com/UMEP-dev/UMEP/files/7801533/POI_1.txt)

Kind regards

**Reply by biglimp** [reply]
Ok, then I can test with the tutorial data.

**Reply by biglimp** [reply]
One question. Are you using Solweig via the Menu plugin or the processing toolbox?

**Reply by danelzw** [reply]
I am not 100 % sure what you mean. This is how it looks at QGIS. Hope that it answers your question :)
<img width="1378" alt="image" src="https://user-images.githubusercontent.com/97020500/147925954-ab402184-24de-4332-a398-6fcb0985478d.png">
<img width="686" alt="image" src="https://user-images.githubusercontent.com/97020500/147925997-c795dd52-8b8c-4a29-980d-65f42a0c6bb5.png">

**Reply by biglimp** [reply]
Yes, THanks. FYI, we also have UMEP for processing which makes it possible to run UMEP plugins from the processing toolbar (https://umep-docs.readthedocs.io/en/latest/UMEPforProcessing.html)

**Reply by biglimp** [reply]
Ok, I think I know what is happening. There is no wind speed (-999.00) in the met-file in the tutorial. That could be why you get strange PET-values and -999 in UTCI. Check the wind speed in your met-data and try again. I will add a warning is no wind speed is present in the plugin for the next version.

**Reply by danelzw** [reply]
Oh, that really can be the case! I also did not use any wind data because in the tutorial its said that one only have to use temp, rh and radiation (see picture) but I guess it is only for the TMRT calculation. I will try it again with the wind speed data, thank you for your help! 
<img width="284" alt="image" src="https://user-images.githubusercontent.com/97020500/147932813-6e6b556e-9539-4c9f-9da4-ebdbbbbe708b.png">

**Reply by biglimp** [reply]
Great. I will also change in the tutorial regarding the use of POIs.

**Reply by danelzw** [reply]
It worked, I get different PET for each POI! Thank you again. 
But I cannot "prepare existing data" with windspeed of 0 m/s. Can you explain why this is the case? 
Kind Regards, 
Dana

**Reply by biglimp** [reply]
Yes. We have models in UMEP that is failing if ws=0. Set to e.g. 0.01 if you have no wind. However, ws of 0 m/s is not probable in outdoor environmnets.

**Reply by danelzw** [reply]
Ok, great! Thank you :)
